NPB3.3-MPI/Dockerfile

```diff
@@ -0,0 +1,53 @@
+#The NBP Benchmark will run on ubuntu
+FROM ubuntu:latest
+
+
+#Get the NPB from http://www.nas.nasa.gov/publications/npb.html 
+RUN apt-get update && apt-get install -y\
+	wget\
+	bc &&\
+	wget http://www.nas.nasa.gov/assets/npb/NPB3.3.1.tar.gz &&\
+	tar -zxvf NPB3.3.1.tar.gz &&\
+	rm -f NPB3.3.1.tar.gz &&\
+	mkdir /Bench &&\
+	mv  NPB3.3.1/NPB3.3-MPI /Bench/. &&\
+	rm -rf NPB3.3.1
+
```

Comment:
Don't forget to use the best practices when you apply such command.
https://docs.docker.com/engine/articles/dockerfile_best-practices/